apiserver/params/model.go

```diff
@@ -139,7 +139,7 @@ type ModelInfo struct {
 	Life Life `json:"life"`
 
 	// Status is the current status of the model.
-	Status EntityStatus `json:"status"`
+	Status *EntityStatus `json:"status,omitempty"`
```

Comment:
Will existing 2.0 or 2.1 clients handle this change?
Actually, this will break the jem codebase. (jem/internal/jujuapi/controller.go)